lib/Common/DataStructures/BaseDictionary.h

```diff
@@ -1239,7 +1239,7 @@ namespace JsUtil
         public:
             EntryIterator(TDictionary &dictionary) : Base(dictionary, 0), entryCount(dictionary.count)
             {
-                if(IsValid() && IsFreeEntry(entries[entryIndex]))
+                if(IsValid() && IsFreeEntry(__super::entries[entryIndex]))
```

Comment:
IIRC this is not an issue with clang as a part of MS compatibility and we already use it project wide.